publish.sh

```diff
@@ -23,8 +23,12 @@ for i in ${CLUSTERS}; do
     if [[ $i =~ ipv6 ]]; then
         NETWORK_STACK="ipv6"
     fi
+
     cluster-provision/gocli/build/cli provision cluster-provision/k8s/$i --network-stack ${NETWORK_STACK}
     docker tag ${TARGET_REPO}/k8s-$i ${TARGET_REPO}/k8s-$i:${KUBEVIRTCI_TAG}
+
+    cluster-provision/gocli/build/cli provision cluster-provision/k8s/$i --network-stack ${NETWORK_STACK} --slim
```

Comment:
This is going to increase the run time for the publish-kubevirtci job which I think has a timeout currently of 3 hours - @oshoval - from your testing does this additional provision add much time? 
https://github.com/kubevirt/project-infra/blob/4f229481a8feb975199efc7ee853534c78aab8af/github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-postsubmits.yaml#L3-L11

Reply:
Atm we stand between 1:20 - 1:50 as we can see here
https://prow.ci.kubevirt.io/?job=publish-kubevirtci

It will add time, but not alot, because the images are small and there is no caching needed
lets say 5m x 4 to create the providers + 3m x 4 to push them, so it adds around 35m

Sometimes the whole lane is slower, and then it might be marginal
lets run it once and then see by how much to extend ?
It should pass according the benchmarks above

Reply:
Merged, ptal